### PR TITLE
XWIIKI-13284 Add support for loading RSS feeds over HTTPS in the RSS macro

### PR DESCRIPTION
**Pull request by seanthegeek**
…Macro

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/DefaultRomeFeedFactory.java [hunk 1]
```diff
@@ -43,6 +44,9 @@
      * The maximum number of milliseconds to wait when inquiring the RSS feed provider.
      */
     private static final int TIMEOUT_MILLISECONDS = 5000;
+    private static final String USER_AGENT_HEADER = "User-Agent";
+    private String version = this.getClass().getPackage().getImplementationVersion();
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-rss/src/main/java/org/xwiki/rendering/internal/macro/rss/DefaultRomeFeedFactory.java]: I feel those fields would make more sens as static, for example:

    private static final String VERSION = DefaultRomeFeedFactory.class.getPackage().getImplementationVersion();

**Comment by seanthegeek** [comment]
Agreed. Fixed.

**Comment by tmortagne** [comment]
Applied, thanks !

Would be great if you could add something about it in http://www.xwiki.org/xwiki/bin/view/ReleaseNotes/ReleaseNotesXWiki81M2 and also document it on xwiki.org.

**Comment by seanthegeek** [comment]
Done

**Pull request merged**